CMakeLists.txt

```diff
@@ -20,25 +22,32 @@ add_compile_options("-Wno-format-security")
 #add_compile_options("-Wstrict-overflow=5")
 #add_compile_options("-Wdisabled-optimization")
 
-enable_testing()
+if (ENABLE_TESTS)
+  enable_testing()
+endif()
 
-if (OFFLINE_BUILDS)
-  include(ExternalProject)
-  ExternalProject_Add(bcc
-    GIT_REPOSITORY https://github.com/iovisor/bcc
-    STEP_TARGETS build update
-    EXCLUDE_FROM_ALL 1
-    UPDATE_DISCONNECTED 1
-    BUILD_COMMAND ${CMAKE_COMMAND} --build . --target bcc-static
-    )
+if (SYSTEM_BCC_LIBRARY)
+  find_package(LibBcc REQUIRED)
+  include_directories(${LIBBCC_INCLUDE_DIRS})
```

Comment:
I think it makes sense to convert the existing `target_include_directories()` calls to use `include_directories()` like this for the cases where we're not using the system BCC as well. Pretty much all the targets need it anyway and having it in the root CMakeLists like this is simpler.

Reply:
Fixed (FYI, I've amended the same commit)